source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -268,7 +268,7 @@
                 contentType: 'application/json; charset=utf-8',
                 data: JSON.stringify({ evaluationId: this.id }),
                 success: function (response) {
-                    getDashboardEvaluations(window.ViewData.period);
+                    window.location.reload();
```

Comment:
he only thing that comes to my mind is that the user tries to add or remove evaluators, and without accepting changes, starts the evaluation devolution. in that case, any change done to the evaluators list will be lost, but it's kind of an illogical action :confused: